scriptworker/task.py

```diff
@@ -70,16 +70,36 @@ def get_run_id(claim_task):
 def get_decision_task_id(task):
     """Given a task dict, return the decision taskId.
 
+    By convention, the decision task of the ``taskId`` is the task's ``taskGroupId``.
+
     Args:
         task (dict): the task dict.
 
     Returns:
-        str: the taskId.
+        str: the taskId of the decision task.
 
     """
     return task['taskGroupId']
```

Comment:
Should we raise a specific error if a decision/action task doesn't reference a taskGroupId?

Reply:
I don't think there's a valid way to get a task without a `taskGroupId`; am I wrong?

Reply:
Oh right, I think that was possible with the Scheduler API, but that's out of scope of CoT.